assets/differentialabundance_report.Rmd

```diff
@@ -895,6 +895,25 @@ for (i in 1:nrow(contrasts)){
         if (nrow(contrast_de) > 0){
             contrast_de <- round_dataframe_columns(contrast_de, digits=params$report_round_digits)
             print( htmltools::tagList(datatable(contrast_de, caption = paste('Differential genes', dir, 'in', contrast_descriptions[i], " (check", differential_files[[i]], "for more detail)"), rownames = FALSE) ))
+
+        # Plot Differentially Expressed Genes by Gene Biotype
+        gene_biotype_table <- contrast_de %>%
+            group_by(`Gene biotype`) %>%
+            dplyr::summarise(count = dplyr::n(), .groups = 'drop') %>%
+            dplyr::filter(count > 0) %>%
+            dplyr::arrange(desc(count))
```

Comment:
I always try to refer explicitly to avoid conflicts, especially when several packages use "filter", but I am happy to hear if there is any other convention/best practice :)

Reply:
Yeah, let's take out the `dplyr`, repetitive I think. Definitely let's not add another dependency!